src/utils/parsers/index.js

```diff
@@ -161,15 +161,19 @@ const utils = {
   },
   handleXMLStringResponseBeforeConvertToJSON({response, layers, wms}={}) {
     if (!response) return; // return undefined if no response
-    if (!(typeof response === 'string'|| response instanceof String))
+    if (!(typeof response === 'string'|| response instanceof String)) {
       response = new XMLSerializer().serializeToString(response);
+    }
     for (let i=0; i < layers.length; i++) {
       const layer = layers[i];
       let originalName = (wms && layer.isWmsUseLayerIds()) ? layer.getId(): layer.getName();
-      let sanitizeLayerName = wms ? originalName.replace(/[/\s]/g, '') : originalName.replace(/[/\s]/g, '_');
-      sanitizeLayerName = sanitizeLayerName.replace(/(\'+)/, '');
-      sanitizeLayerName = sanitizeLayerName.replace(/(\)+)/, '');
-      sanitizeLayerName = sanitizeLayerName.replace(/(\(+)/, '');
+      let sanitizeLayerName = wms ? originalName.replace(/[/\s]/g, '') : originalName.replace(/\s+/g, '_');
+      sanitizeLayerName     = sanitizeLayerName.replace(/(\'+)/, '');
+      sanitizeLayerName     = sanitizeLayerName.replace(/(\)+)/, '');
+      sanitizeLayerName     = sanitizeLayerName.replace(/(\(+)/, '');
+      sanitizeLayerName     = wms ? sanitizeLayerName : sanitizeLayerName.replace(/\/+/g, '');
```

Comment:
@volterra79 I think that by using the "+" modifier may result in rare cases of collisions:

| ORIGINAL | SANITIZED |
|-------|-------|
| `layername`    | `layername` |
| `layer / name` | `layername` |
| `layer/ /name` | `layername` |

If so, let's try to test them directly in here: https://github.com/g3w-suite/g3w-client/pull/541

Reply:
@Raruto try rename building with **buildings: wfs (a/z)**. Qgis server turn as layername **buildings-_wfs_az**.
It replace : with -, blank space with _, remove ( ) and / with empty space